crates/sui-framework/packages/sui-framework/sources/object.move

```diff
@@ -43,6 +43,9 @@ const SUI_DENY_LIST_OBJECT_ID: address = @0x403;
 /// The hardcoded ID for the Bridge Object.
 const SUI_BRIDGE_ID: address = @0x9;
 
+/// The hardcoded ID for the Coin Metadata Registry Object.
+const SUI_COIN_METADATA_OBJECT_ID: address = @0x10;
```

Comment:
Note that `0xa` is the next address after `0x9`, but maybe you want to use `0xc` for the coin registry, and `0xd` for the display registry? (`0xb` is already taken by the bridge package so `0xa` and `0xb` is not an option...)